.githooks/pre-push

```diff
@@ -1,3 +1,19 @@
 #!/bin/sh
 command -v git-lfs >/dev/null 2>&1 || { echo >&2 "\nThis repository is configured for Git LFS but 'git-lfs' was not found on your path. If you no longer wish to use Git LFS, remove this hook by deleting .git/hooks/pre-push.\n"; exit 2; }
 git lfs pre-push "$@"
+
+// Linting the frontend
+git diff HEAD^ --exit-code frontend > /dev/null
+if [ $? -eq 1 ]; then
+	cd frontend
+	npx eslint "{src,apps,libs,test}/**/*.ts" --fix --max-warnings=0 || exit 1
```

Comment:
Is it intentional to do --fix here instead and --no-fix on the backend?

Reply:
No, thank you !